client/src/sass/overrides.sass

```diff
@@ -58,7 +58,10 @@
   background-color: $link !important
   height: 40px !important
   padding: inherit 28px inherit 28px !important
+  border-bottom-width: 2px !important
```

Comment:
This messes with blue-on-white buttons (which is why I put in a SCSS override to set this to 1px). What's the reason for this? Should this be a drop shadow instead?

Reply:
I've no idea.. did i put this in? 😆 
Removed, but i'll research if there was a specific use for this.

Reply:
Ok, i must have but i'm not sure why. Removed 👍